classes/usage/quota_manager.php

```diff
@@ -229,6 +247,16 @@ protected function notification_needs_sending($notificationtype, $settings) {
             return true;
         }
 
+        //P.T. 29-ß9-2022 added notificationtype conditions for new option of disabled automatic side blockage
+        if ($notificationtype == 'over_quota_noblock') {
+            return true;
+        }
+
+        //P.T. 29-ß9-2022 added notificationtype conditions for new option of disabled automatic side blockage
+        if ($notificationtype == 'nearing_quota_noblock') {
+            return true;
+        }
+
```

Comment:
Doesn't this mean that it will spam users with warning since the frequency is ignored?